legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/metamodel/test/relationalTest.pure

```diff
@@ -0,0 +1,53 @@
+// Copyright 2022 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// ------------------------------------------------------------------------------ ASSERTION -----------------------------------------------
+
+Class meta::relational::metamodel::assertion::EqualToTDS extends meta::pure::test::assertion::TestAssertion
+{
+ expected: meta::relational::metamodel::assertion::RelationalTDS[1];
+}
+
+Class  meta::relational::metamodel::assertion::RelationalTDS
+{
+  columns:  meta::relational::metamodel::data::RelationalDataTableColumn[*];
+  rows: meta::relational::metamodel::data::RelationalDataTableRow[*];
+}
+
+// ------------------------------------------------------------------------------ DATA -----------------------------------------------
+Class meta::relational::metamodel::data::RelationalData extends  meta::pure::data::EmbeddedData
+{
+  tables: meta::relational::metamodel::data::RelationalDataTable[*];
+}
+
+Class meta::relational::metamodel::data::RelationalDataTable
+{
+  schemaName: String[0..1];
+  tableName: String[1];
+  columns: meta::relational::metamodel::data::RelationalDataTableColumn[*];
+  rows: meta::relational::metamodel::data::RelationalDataTableRow[*];
+}
+
+Class meta::relational::metamodel::data::RelationalDataTableRow
+{
+  values: String[1];
```

Comment:
should we do the parsing of row values in the parser step itself too so that we have a list of values of rows, this might be helpful as parsing in java would be easier

Reply:
moved this just to csv.